addons/website_event/views/event_templates.xml

```diff
@@ -206,7 +206,7 @@
                     <span class="navbar-toggler-icon"></span>
                 </button>
                 <div class="collapse navbar-collapse" id="o_wevent_navbar_event_content">
-                    <ul class="navbar-nav">
+                    <ul class="navbar-nav" name="Event Menu" t-att-data-content_menu_id="editable and event.menu_id.id">
```

Comment:
The PR did change, you added `editable` :p In fact `name` is not a valid attribute for a `<ul/>`, could use `data-name` and adapt the related JS if needed ?

Reply:
Not in stable

Reply:
Added fallback

Reply:
Reminder ;-) this should be good